### PR TITLE
Update Mac OS dev instructions

### PR DESCRIPTION
**Pull request by abulte**
Homebrew changed where it stores its files.

### CODE REVIEW

**Review by quaxsze — APPROVED** [review]

File: docs/development-environment.md [hunk 1]
```diff
@@ -38,24 +38,25 @@ $ pip install -r requirements/develop.pip
...
-### Macos Big Sur caveat
+### Mac OS caveat
 
 If installing `cryptography` fails:
 
 ```
 brew install openssl
-export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
-export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
+export LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib"
```
Comment (nicolaskempf57) [inline docs/development-environment.md:47]:
```suggestion
export LDFLAGS="-L$(brew --prefix openssl)/lib"
```

File: docs/development-environment.md [hunk 2]
```diff
@@ -38,24 +38,25 @@ $ pip install -r requirements/develop.pip
...
+export CPPFLAGS="-I/opt/homebrew/opt/openssl@1.1/include"
```
Comment (nicolaskempf57) [inline docs/development-environment.md:48]:
```suggestion
export CPPFLAGS="-I$(brew --prefix openssl)/include"
```

**Review by nicolaskempf57 — COMMENTED** [review]
Homebrew uses different paths based on the OS and arch.

We can use `brew --prefix` to get the correct one.

File: docs/development-environment.md [hunk 3]
```diff
@@ -38,24 +38,25 @@ $ pip install -r requirements/develop.pip
 $ pip install -e .
 ```
 
-### Macos Big Sur caveat
+### Mac OS caveat
 
 If installing `cryptography` fails:
 
 ```
 brew install openssl
```
Comment (nicolaskempf57) [inline docs/development-environment.md:46]: We should specify openssl@1.1 here too.

For reference, udata is using [cryptography 2.8](https://cryptography.io/en/2.8/installation/) not tested with openssl@3.

```suggestion
brew install openssl@1.1
```

Comment (abulte) [reply docs/development-environment.md:46]: Yep, done in latest commit ;-)

Comment (nicolaskempf57) [reply docs/development-environment.md:46]: Is it pushed already ? I'm not seeing it for this line.

Comment (abulte) [reply docs/development-environment.md]: Yeah, right, sorry.

Comment (nicolaskempf57) [reply docs/development-environment.md]: Thanks ! 🙏🏻

**Review by nicolaskempf57 — APPROVED** [review]

**Pull request merged**